_inc/client/searchable-modules/index.jsx

```diff
@@ -46,7 +46,6 @@ export const SearchableModules = moduleSettingsForm(
 				'enhanced-distribution',
 				'json-api',
 				'latex',
-				'monitor',
```

Comment:
This list is: _"Only should be features that don't already have a UI, and we want to reveal in search._"

Monitor now has UI.